plugins/scaffolder-backend/.eslintrc.js

```diff
@@ -1,41 +1,18 @@
-const parent = require('@backstage/cli/config/eslint.backend');
-
-module.exports = {
-  extends: [require.resolve('@backstage/cli/config/eslint.backend')],
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
```

Comment:
This looks a lot cleaner! 👍 

In a future world when I want all the default rules except for X, is there a way of doing that our do I have to opt out?

Reply:
Yep you can disable rules by setting them to `'off'` or `0`

Reply:
Realize that if you meant the restricted imports etc. then it's only possible to add atm. You can completely replace the rule configuration as well though